src/components/buttons/OutlinedButton.js

```diff
@@ -0,0 +1,46 @@
+import {Color, Typography} from '../../themes';
+import {ColorVariant} from '../../themes/color';
+import React from 'react';
+import ButtonItem from './buttonItem/ButtonItem';
+
+export default function OutlinedButton(props) {
+  const {
+    content,
+    message,
+    colorText = ColorVariant.primary,
+    colorOutline = ColorVariant.outline,
+    colorSurface = ColorVariant.surface,
+    typography = Typography.label.large,
+    children,
+    style,
+    ...otherProps
+  } = props;
+
+  const {base: baseText} = Color.light[colorText];
+  const {base: baseOutline} = Color.light[colorOutline];
+  const {base: baseBackground} = Color.light[colorSurface];
+
+  const containerStyle = [
+    {
+      borderColor: baseOutline,
+      borderWidth: 1,
+      backgroundColor: baseBackground,
+    },
+    style,
+  ];
+
+  const textStyle = [{color: baseText}, typography, style];
+  return (
+    <ButtonItem
+      {...otherProps}
+      style={{
```

Comment:
we should split the props "style" into two props style, textStyle , not one Object style only to make it simpler and more natural